tools/web3-plugin-example/package.json

```diff
@@ -45,10 +45,10 @@
 		"prettier": "^2.7.1",
 		"ts-jest": "^28.0.7",
 		"typescript": "^4.7.4",
-		"web3": "^4.2.0",
+		"web3": "^4.2.1",
```

Comment:
you will need to bump patch of plugin example if modifying its package.json, also its change log